products/accesscontextmanager/api.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Access Context Manager
+prefix: gaccesscontextmanager
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://accesscontextmanager.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+objects:
+  - !ruby/object:Api::Resource
+    name: 'AccessPolicy'
+    base_url: accessPolicies
+    exclude: true
```

Comment:
I don't _think_ you need this exclude, since the lack of a [provider].yaml file for the others will mean it doesn't get generated.